### DISCUSSION TITLE
I have a Nyth. What can I do to help get full support?

### DISCUSSION THREAD

**Discussion by PadreAdamo** [X3n0m0rph59/eruption#115] [Hardware Support]
I have a Nyth. I'm not a programmer but I have been using Linux for 10 years. What do you need me to do to assist with getting full support?

Thank you for all your hard work.

-P

**Comment by X3n0m0rph59** [comment]
Hi @PadreAdamo, thank you for offering your assistance!

To assist in improving the support for the `ROCCAT Nyth` you would need to build `Eruption` from source multiple times to try out new patches. I have added a `Makefile` very recently that automates most of the required tasks, so it is not very complicated.

[HowTo install from source](https://github.com/X3n0m0rph59/eruption/blob/master/docs/INSTALL.md)
(Please remove the packages `eruption` or `eruption-git` from your system before building/installing from source)

There exists a tracking issue for the `ROCCAT Nyth` (issue #40)

**Comment by X3n0m0rph59** [comment]
@PadreAdamo 

The first steps would be:

* Update your `ROCCAT Nyth` to the latest firmware revision using `ROCCAT Swarm` on Windows 
* Reset your `ROCCAT Nyth` to factory defaults using `ROCCAT Swarm` on Windows 

Since I don't have access to that hardware, I need to know what is currently working and what not.
Could you please post the output of the `eruption` daemon running with `debug` log level:

```sh
sudo RUST_LOG=debug RUST_BACKTRACE=full eruption
```
